gapis/server/grpc.go

```diff
@@ -330,50 +330,51 @@ func (s *grpcServer) Profile(stream service.Gapid_ProfileServer) error {
 	}
 }
 
-func (s *grpcServer) Status(stream service.Gapid_StatusServer) error {
+func (s *grpcServer) Status(req *service.ServerStatusRequest, stream service.Gapid_StatusServer) error {
 	defer s.inRPC()()
```

Comment:
This is what causes gapis to not die. It should be removed. In the log stream RPC I put in a comment why it's not there, so it won't get added in later, since not having it is the "special case".

Reply:
Done